installer_windows.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# make protobufs for logging part first
+make all
+
+# get tensorflow
+git clone https://github.com/tensorflow/tensorflow
+cd tensorflow
+
+# run configuration.
+bash configure --prefix=/mingw

```

Comment:
This the only thing worked with me. Without this, I got bazel build errors.
See [Reference](http://www.mingw.org/wiki/msys)